docs/characterization/xeb_theory.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>are we assuming people have seaborn installed? I guess on colab this is true. We should figure out how we handle requirements for notebooks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>it's in <code>notebook_test.py</code>  as an installed package / requirement. I assumed other notebooks were therefore using it</p>

Reply:
Oh, you're right. I added those there as an assumption that colab has seaborn installed.